app/src/main/res/layout/activity_movie_details.xml

```diff
@@ -0,0 +1,180 @@
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:layout_toRightOf="@id/movie_poster"
+                        android:layout_toEndOf="@id/movie_poster"
+                        android:orientation="vertical">
+
+                        <TextView
+                            android:id="@+id/movie_name"
+                            android:layout_width="match_parent"
+                            android:layout_height="wrap_content"
+                            android:textStyle="bold"
+                            android:textSize="16sp"
+                            android:textColor="@color/color_black"
+                            tools:text=""/>
+
+                        <TextView
+                            android:id="@+id/movie_genres"
+                            android:layout_width="match_parent"
+                            android:layout_height="wrap_content"
+                            android:layout_marginTop="@dimen/horizontal_padding"
+                            tools:text=""/>
+
+                        <TextView
+                            android:id="@+id/movie_duration"
+                            android:layout_width="match_parent"
+                            android:layout_height="wrap_content"
+                            android:paddingTop="@dimen/horizontal_padding"
+                            tools:text=""/>
+
+                        <TextView
+                            android:id="@+id/movie_tagline"
+                            android:layout_width="match_parent"
+                            android:layout_height="wrap_content"
+                            android:textStyle="italic"
+                            android:textSize="12sp"
+                            android:layout_marginTop="@dimen/horizontal_padding"
+                            tools:text=""/>
+                    </LinearLayout>
+
+                </RelativeLayout>
+
+                <View
+                    android:layout_width="match_parent"
+                    android:layout_height="1dp"
+                    android:background="@color/divider"/>
+
+                <TextView
+                    android:id="@+id/movie_overview"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:padding="@dimen/activity_vertical_margin"
+                    android:textColor="@color/color_black"
+                    tools:text=""/>
+
+                <View
+                    android:layout_width="match_parent"
+                    android:layout_height="1dp"
+                    android:background="@color/divider"/>
+
+                <TextView
+                    android:layout_width="match_parent"
+                    android:layout_height="50dp"
+                    android:gravity="center_vertical"
+                    android:background="@color/white"
```

Comment:
You could have added a background with a `shape` drawing the 2 lines (top and bottom) instead of using 2 views.